conbench/api/history.py

```diff
@@ -37,8 +26,19 @@ def get(self, benchmark_id):
         tags:
           - History
         """
-        history = self._get(benchmark_id)
-        return self.serializer.many.dump(history)
+        # TODO: think about the case where samples if of zero length. Can this
+        # happen? If it can happen: which response would we want to emit to the
+        # HTTP client? An empty array, or something more convenient?
+        try:
+            samples = get_history_for_benchmark(benchmark_result_id=benchmark_id)
```

Comment:
In this PR I was sneaking in a clarification from "benchmark ID" towards "benchmark result ID". 

It should be an ongoing effort to make the code base clearer in that regard.